### PR TITLE
docs: Fix Compose documentation

### PR DESCRIPTION
**Pull request by mnapoli**
Last minute fixes to the docs.

### CODE REVIEW

File: docs/guides/compose.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ services:
 
   service-b:
     path: service-b
-    # If the file is not named "serverless.yml" it is possible to configure that:
+    # If the file is not named "serverless.yml":
```
Comment (mnapoli) [inline docs/guides/compose.md:53]: It didn't fit the page width so I shortened it.

File: docs/guides/compose.md [hunk 2]
```diff
@@ -168,9 +168,9 @@ $ serverless logs --tail
 service-a › users › START
 service-a › users › 2021-12-31 16:54:14  INFO  New user created
 service-a › users › END Duration: 13 ms ...
-service-b › subscriptions › START
-service-b › subscriptions › 2021-12-31 16:54:14  INFO  New subscription enabled
-service-b › subscriptions › END Duration: 7 ms ...
+service-b › billing › START
+service-b › billing › 2021-12-31 16:54:14  INFO  New subscription enabled
+service-b › billing › END Duration: 7 ms ...
```
Comment (mnapoli) [inline docs/guides/compose.md:173]: Same here, shorter name to fit the width of the page.

**Review by pgrzesik — APPROVED** [review]
👍

**Comment by mnapoli** [comment]
@pgrzesik any idea why the last check is stuck?

**Comment by pgrzesik** [comment]
@mnapoli Snyk tends to get stuck from time to time - feel free to merge

**Pull request merged**